server/src/main/java/com/coincoinche/engine/cards/Rank.java

```diff
@@ -2,11 +2,6 @@
 
 /** Rank is an enum representing a card's rank. */
 public enum Rank {
-  TWO("2"),
-  THREE("3"),
-  FOUR("4"),
-  FIVE("5"),
-  SIX("6"),
```

Comment:
We had a 52 card deck, I did not notice before ^^

Reply:
Haha indeed!